pyartcd/pyartcd/pipelines/build_oadp.py

```
@@ -154,29 +155,23 @@ async def _rebase_and_build(self):
-            if not kubeconfig:
-                available_env_vars = list(GROUP_KUBECONFIG_MAP.values())
-                raise ValueError(
-                    f"Kubeconfig required for Konflux builds. Provide --kubeconfig parameter or set one of: {', '.join(available_env_vars)}"
-                )
+        # Use kubeconfig from CLI parameter or resolve from product-specific environment variable
+        kubeconfig = resolve_konflux_kubeconfig_by_product(product, self.kubeconfig)
+        if not kubeconfig:
+            available_env_vars = list(PRODUCT_KUBECONFIG_MAP.values())
+            raise ValueError(
+                f"Kubeconfig required for Konflux builds. Provide --kubeconfig parameter or set one of: {', '.join(available_env_vars)}"
+            )
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)